pages/article-devt-process.html

```diff
@@ -0,0 +1,143 @@
+	</header>
+<h1>Article development process</h1>
+
+<section id="sidebarExtras">
+</section>
+
+
+<div id="mainLayout">
+<section>
+  <p>This page describes stages in article development within the Internationalization Working Group. Not every step is required. It is intended more as a guide. There are however points where group agreement is needed to move forward, and these are required.</p>
+
+  <p>This is provided for new group members to become familiar with the way we work.</p>
+</section>
+
+
+
+
+
+<section id="intent">
+    <h2>Stage: Intent</h2>
+
+    <p class="begin"><strong>This stage begins when:</strong> <br>An author indicates that they intend to create an article, on the WG mailing list or during a teleconference.</p>
+		<p>For FAQs, authors are encouraged to send out the wording of the proposed question for suggestions from the group.</p>
+</section>
+
+<section id="brainstorm">
+  <h2>Stage: Brainstorm</h2>
+
+  <p class="begin"><strong>This stage begins when:</strong> <br>The author produces some questions or a proposed list of topics to be covered by
+    the material, and asks for comments.</p>
+  <p>After making a first stab at an article, the author may:</p>
+  <ul>
+    <li>review ideas with the chair/team contact</li>
+    <li>request that WG participants help in brainstorming to frame a topic, or discussing ideas that are still needing development. This may be
+     done in email or by telecon. Preferably, email would precede telecon discussions. The author should ask the team contact to schedule a discussion for
+     a telecon.</li>
+  </ul>
+</section>
+
+<section id="initial">
+  <h2>Stage: Initial draft</h2>
+  <p class="begin"><strong>This stage begins when:</strong> <br>The author produces a first draft of the proposed text and announces it on the
+    WG mailing list, asking for comments.</p>
+  <p>This first review stage is still to allow exploration of direction and scoping for the author. The author should be ready to substantially
+    modify their draft based on comments received.</p>
+  <p>WG participants may comment in email prior to any telecon discussions. Telecons should be used for discussion of points that are
+    difficult to handle in email. [Note to telecon participants: it is not good form to discuss a draft at the telecon without having at least read the
+    draft and thought about it in advance.]</p>
+  <p>Comments on small editorial nits are likely to be less relevant at this stage. </p>
+</section>
+
+<section id="wd">
+  <h2>Stage: Working draft</h2>
+  <p class="begin"><strong>This stage begins when:</strong> <br>The author produces a new draft, incorporating all appropriate comments received on the
+    initial draft, and sends a call for comments on the WG mailing list.</p>
+  <p>The author should try to make this as near as possible to publication ready. See the
+    <a href="https://w3c.github.io/i18n-activity/guidelines/editing">editorial guidelines for i18n docs</a>.</p> 
```

Comment:
just a question, better to use w3.org URL or not? For this, https://www.w3.org/International/i18n-activity/guidelines/editing

Reply:
Thanks. Should be fixed now.